package.json

```diff
@@ -34,9 +34,9 @@
     "sinon": "^7.4.2"
   },
   "dependencies": {
-    "aws-sdk": "^2.224.1",
+    "aws-sdk": "^2.804.0",
     "chalk": "^2.3.2",
-    "lodash.get": "^4.4.2"
```

Comment:
isn't it better to load smaller lodash.get package rather than installing complete lodash ?

Reply:
@rrahul963 Yeah, it definitely makes sense. I returned "lodash.get " back.